packages/editor-ui/src/components/canvas/elements/nodes/CanvasNodeToolbar.vue

```diff
@@ -1,12 +1,16 @@
 <script setup lang="ts">
 import { computed, inject, useCssModule } from 'vue';
 import { CanvasNodeKey } from '@/constants';
+import { useI18n } from '@/composables/useI18n';
+
+const emit = defineEmits(['delete']);
+const $style = useCssModule();
 
 const node = inject(CanvasNodeKey);
```

Comment:
It's unrelated to these PR changes, but honestly I'm not a huge fan of relying on provide/inject patterns so heavily. Especially since the props we're passing are only 1/2 level deep. And the lines we save are negated anyway because we have to do stuff like this: 
```ts
const label = computed(() => handle?.label.value ?? '');
``` 
or
```ts
const label = computed(() => node?.label.value ?? '');
const inputs = computed(() => node?.data.value.inputs ?? []);
const outputs = computed(() => node?.data.value.outputs ?? []);
```

IMO, having a utility function to generate Node props would be cleaner

Reply:
Was afraid we'd end up having super deep prop-drilling, which is why I opted for provide/inject. If you feel like doing this, I created a task for refactoring it: https://linear.app/n8n/issue/N8N-7429/improve-passing-down-props-to-canvas-node-elements. We can then asses how it looks.